src/components/Perfil/usuario/PerfilUsuario.js

```diff
@@ -70,7 +70,16 @@ export default function PerfilUsuario() {
       setRating(user.rating);
       axiosHeader.get("/users/id/"+user.userId)
       .then(function (response){
-        setImage("http://localhost:8080/api/files/"+user.userId+"/"+response.data.imageFileURL);
+        if (response.data.imageFileURL != "" || response.data.imageFileURL != null){
+          axiosHeader.get("/files/"+user.userId+"/"+response.data.imageFileURL , {responseType: "arraybuffer"})
```

Comment:
The images and other assets can also be stored in a storage provider (e.g. AWS S3) and instead of downloading it from your backend you can return a signed URL from the provider to let the frontend go there and retrieve the asset.